src/components/SpeciesDeliverableTable/index.tsx

```diff
@@ -66,8 +64,7 @@ const SpeciesDeliverableTable = ({ deliverable }: SpeciesDeliverableTableProps):
   };
 
   const onAcceleratorSpeciesClick = (row: any) => {
-    setCurrentParticipantProjectSpecies(row.id);
-    goToParticipantProjectSpecies(deliverable.id, row.projectId, row.speciesId);
+    goToParticipantProjectSpecies(deliverable.id, row.projectId, row.id);
```

Comment:
I thought about doing this, setting the participantProjectSpecies in the url, instead of the speciesId. I think we can change the constant to be: `/accelerator/deliverables/:deliverableId/submissions/:projectId/ppSpecies/:ppSpeciesId` to be clear we are passing the ppSpeciesId instead of the speciesId